cpp/tests/rolling/grouped_rolling_test.cpp

```diff
@@ -638,6 +638,29 @@ TYPED_TEST(GroupedRollingTest, ZeroWindow)
     grouping_keys, input, expected_group_offsets, preceding_window, following_window, 1);
 }
 
+using GroupedRollingTestInts = GroupedRollingTest<int32_t>;
+
+TEST_F(GroupedRollingTestInts, SumLargeWindow)
+{
+  fixed_width_column_wrapper<int32_t, int32_t> input({1, 1, 1, 1, 1, 1, 1, 1, 1, 1});
+
+  size_type preceding_window = 2147483640;
+  size_type following_window = 2147483642;
+
+  cudf::table_view groupby_keys;
```

Comment:
is this intended to be empty?

Reply:
Yes this is to match the exact test case where we ran into a problem. After looking at the code closer, it ends up becoming a pass-through to the non-grouped rolling API. So I am happy to check it if you want me to.

Reply:
Sounds good. This case is same as plain rolling window.